.gitignore

```diff
@@ -2,6 +2,7 @@
 **/blocks/
 **/*.rs.bk
 **/rusty-tags.vi
+/**/Cargo.lock
```

Comment:
That could break `cargo chef` in the `iroha2-dev` docker file. Are you sure you want to exclude it?